dashboard/management/commands/createkeys.py

```diff
@@ -0,0 +1,65 @@
+import json
+import os
+from datetime import datetime
+
+from Crypto.PublicKey import RSA
+from django.core.management.base import BaseCommand, CommandParser
+from jwcrypto.jwk import JWK
+
+
+class Command(BaseCommand):
+    privateKeyFileSuffix = '_private.pem'
+    publicKeyFileSuffix = '_public.pem'
+    jwkFileSuffix = '_public-jwk.json'
+    baseNameOption = 'basename'
+    timestampFormat = '%Y%m%d%H%M%S'
```

Comment:
Might be able to get away with minute precision. File names are pretty long.

Reply:
±2 characters won't make much difference in the name.  At least during testing, I can easily make multiple key sets within a minute.  Besides, the user can specify a pithy basename to use instead of the timestamp.